### ISSUE TITLE
cannot print output file

### ISSUE THREAD

**Issue by mariaangelesjs** [harryjubb/arpeggio#13]
Hi Harry,
I've been trying to use the print script on python ,but I can't get the *.contacts output file. 
The script runs perfectly fine , but this is all I'm getting : 
INFO//15:09:10.727//Program begin.
INFO//15:09:10.776//Loaded PDB structure (BioPython)
INFO//15:09:10.777//Detected that the input structure contains hydrogens. Hydrogen addition will be skipped.
INFO//15:09:10.984//Loaded PDB structure (OpenBabel)
INFO//15:09:10.994//Mapped OB to BioPython atoms and vice-versa.
INFO//15:09:11.127//Typed atoms.
INFO//15:09:11.137//Determined atom explicit and implicit valences, bond orders, atomic numbers, formal charge and number of bound hydrogens.
INFO//15:09:11.171//Initialised SIFts.
WARNING//15:09:11.174//Chain termini could not be determined for chain E. It may not be a polypeptide chain.
INFO//15:09:11.176//Determined polypeptide residues, chain breaks, termini
INFO//15:09:11.341//Percieved and stored rings.
INFO//15:09:11.361//Perceived and stored amide groups.
INFO//15:09:11.367//Added hydrogens to BioPython atoms.
INFO//15:09:11.372//Added VdW radii.
INFO//15:09:11.377//Added covalent radii.
INFO//15:09:11.381//Completed NeighborSearch.
INFO//15:09:11.384//Assigned rings to residues.
INFO//15:09:11.386//Made selection.
INFO//15:09:11.482//Expanded to binding site.
INFO//15:09:11.483//Flagged selection rings.
INFO//15:09:11.484//Completed new NeighbourSearch.
INFO//15:09:11.553//Calculated pairwise contacts.
INFO//15:09:11.710//Program End. Maximum memory usage was 83.724 MB.
Kind regards,
Maria

**Comment by mariaangelesjs** [comment]
Hi Harry,
I've tried the print command on python and >.text to create an output file, am I missing any command ? 
This is the code I'm running : 
docker run --rm -v :\directory :/file -it arpeggio python arpeggio.py /file.pdb -s RESNAME:XXX -v
As you can read above, the output files should be storing in C. Is it because I'm not making the code only readable by User? I've tried adding -u Guest:Users and docker couldn't recognise that.
Apologies for this, but is my first time using Docker and I'm translating everything for Windows.

Kind regards,
Maria

**Comment by mariaangelesjs** [comment]
I've changed the code to fit with yours and I'm working on Ubuntu subsystem, but this is what happens now:
 docker  -H localhost:2375 run --rm -v c:/:/ -u xxx:xxx -it harryjubb/arpeggio python arpeggio.py /file.pdb -s RESNAME:xxx -v
INFO//09:52:23.183//Program begin.
INFO//09:52:23.221//Loaded PDB structure (BioPython)
INFO//09:52:23.222//Detected that the input structure contains hydrogens. Hydrogen addition will be skipped.
INFO//09:52:23.438//Loaded PDB structure (OpenBabel)
INFO//09:52:23.450//Mapped OB to BioPython atoms and vice-versa.
Traceback (most recent call last):
  File "arpeggio.py", line 861, in <module>
    with open(pdb_filename.replace('.pdb', '.atomtypes'), 'wb') as fo:
IOError: [Errno 13] Permission denied: '/Cb2HU210_fixed6.atomtypes'

**Comment by harryjubb** [comment]
Hi Maria,

Did you resolve your issue? I am away this week and not able to test anything out.

If you are using Ubuntu on WSL, the instructions in the README should work, though I'm not sure how it works if connecting to the Docker daemon run by Windows, particularly with respect to the file paths for the bind mount and the user/group IDs.

In the first instance, running only on Windows, did you get any output files at all?

I may be slow to reply this week but will try to help.

By the way, if you enclose your console output with three backtick (`\``) characters before and after, it will format them as a code block, making them easier to read 🙂 (GitHub supports most [Markdown](https://guides.github.com/features/mastering-markdown/) syntax).

**Comment by mariaangelesjs** [comment]
Hi Harry, 
No worries, it is my first time working with Docker so I had a go with different commands. Yes, I was able to to run arpeggio using PowerShell in User. Ubuntu came out with the same result, but adding ``` "($pwd)" ``` was not possible. 

This is what worked for me:
`` docker run --rm -v :/<host-directory>:/<container-path> -u uid:gid -it arpeggio python arpeggio.py /container-path/file.pdb -s RESNAME:xxx -v ``

I've got all the output files on my container's folder :)
Kind regards,
Maria
